### ISSUE TITLE
Augmented Reality Feature

### ISSUE THREAD

**Issue by pabtab** [blackuy/react-native-twilio-video-webrtc#509]
Hey guys, do we have support for this feature that Its already on Twilio video call:
https://www.twilio.com/blog/2017/10/streem-twilio-video-ar-sync.html

**Comment by slycoder** [comment]
I don't believe this feature is implemented yet in this repo.  If you're willing to lend a hand, it would be appreciated!
